docs: update codemod commands for Express 5 migration guide (#2143)
expressjs/expressjs.com: en/guide/migrating-5.md

Run the following command for run all the codemods available:

```sh

> npx codemod@latest @expressjs/v5/migration-recipe 

Author: ```suggestion
npx codemod@latest @expressjs/v5-migration-recipe
```

```

If you want to run a specific codemod, you can run the following command:

```sh
npx codemod@latest @expressjs/name-of-the-codemod
```

> You can find the list of available codemods [here](https://github.com/expressjs/codemod?tab=readme-ov-file#available-codemods).

Author: ```suggestion
You can find the list of available codemods [here](https://codemod.link/express).
```

You can replace the deprecated signatures with the following command:

```plain-text

> npx codemod@latest @expressjs/req-param

Author: ```suggestion
npx codemod@latest @expressjs/explicit-request-params
```
